doc/radosgw/metrics.rst

```diff
@@ -152,7 +152,9 @@ Op metrics can also be tracked per-user or per-bucket. These metrics are exporte
         ...
     ]
 
-:ref:`rgw-multitenancy` allows the use of buckets and users of the same name simultaneously. If a user or bucket lies under a tenant, a label for tenant in the form ``tenant = {tenantid}`` is added to the metric.
+:ref:`rgw-multitenancy` allows the use of buckets and users of the same name
+simultaneously. If a user or bucket lies under a tenant, a label for the tenant
```

Comment:
with the same name, if they are created under different tenants.

Reply:
https://github.com/ceph/ceph/pull/63782